scripts/neighbor_advertiser

```diff
@@ -281,28 +260,48 @@ def construct_neighbor_advertiser_slice():
 
     all_vlan_interfaces = get_vlan_interfaces()
 
+    vlan_intf_table = config_db.get_table('VLAN_INTERFACE')
+
+    vxlanPort = None
+    if 'vxlan_port' in appl_db.get_all(appl_db.APPL_DB, 'SWITCH_TABLE:switch'):
+        vxlanPort = appl_db.get(appl_db.APPL_DB, 'SWITCH_TABLE:switch', 'vxlan_port')
```

Comment:
This code block looks not efficient. Would appl_db.get() return None when key not existing?